java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -926,6 +927,21 @@ public T scard(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Checks whether each member is contained in the members of the set stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/smismember/">redis.io</a> for details.
+     * @param key The key of the set to check.
+     * @param members A list of members to check for existence in the set.
+     * @return Command response - An <code>array</code> of <code>boolean</code> values, each
```

Comment:
Just a very minor note, most docs in this file capitalize the R in "Response", however there are a handful of existing occurrences that do not